main.js

```diff
@@ -46,6 +48,14 @@ const animate = () => {
   drawPaddles();
   drawBall();
   updatePaddles();
+  ball.x += ballSpeedHorizontal;
```

Comment:
This should be in the ball class `update` method.

This animation function is for high level coordination processing a frame. 

Including low lever details like increment a number is a different level of abstraction then `drawBall`. This animate function should only tell the ball to update.